### DISCUSSION TITLE
Add documentation for mdDown

### DISCUSSION THREAD

**Discussion by lilouartz** [chakra-ui/panda#2700] [General]
### Subject

Breakpoints

### Description

I noticed that I am using an undocumented feature. I cannot find anywhere in the documentation `${breakpoint}Down` mentioned, but it is working, and I have not a clue where I picked it up from.

**Comment by anubra266** [comment]
@lilouartz Feel free to open a PR.
It'll be somewhere around here
https://panda-css.com/docs/concepts/responsive-design#targeting-a-breakpoint-range

**Reply by aleclarson** [reply]
This still isn't documented? 😅
